frontend/packages/knative-plugin/src/components/modals/index.ts

```diff
@@ -0,0 +1,4 @@
+export const setTrafficDistributionModal = (props) =>
+  import(
+    '../traffic-splitting/TrafficSplittingController' /* webpackChunkName: "set-traffic-splitting" */
+  ).then((m) => m.trafficModalLauncher(props));
```

Comment:
We already launch a modal from the sidebar and it isn't lazy loaded.
This probably will cause the code to be in multiple chunks.

Reply:
changed the code to lazy load the model from the sidebar as well